app/src/main/res/values/strings.xml

```diff
@@ -320,6 +320,7 @@
     <string name="two_factors_otp_error">Two factors OTP is required</string>
     <string name="no_issue">No issue</string>
     <string name="copy">Copy</string>
+    <string name="copy_repo_url">Copy repo URL</string>
```

Comment:
I changed is since it makes more sense thus way

If you want I can change it back

_Sent from my HTC Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_